.github/workflows/deploy-full-stack.yml

```
@@ -0,0 +1,406 @@
+          retention-days: 7
+
+  # Deployment Summary
+  deployment-summary:
+    name: Deployment Summary
+    runs-on: ubuntu-latest
+    needs: [deploy-backend, deploy-frontend, integration-tests]
+    if: always()
+
+    steps:
+      - name: Create deployment summary
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)